graderx/graders/manager.py

```diff
@@ -153,32 +153,32 @@ def get_all_labs_data(course_id):
         course_labs[index]["test_cases"] = stdout_common.get_test_cases(course_id, lab["name"])
     return course_labs
 
-def get_submission_files(course, lab, submission_id):
+def get_submission_files(course, lab, submission_id, submission_key):
     course_grader = select_course_grader(course)
-    submission_files_paths = course_grader.get_submission_files(course, lab, submission_id)
+    submission_files_paths = course_grader.get_submission_files(course, lab, submission_id, submission_key)
     if submission_files_paths:
         return list(map(lambda path: path.name, submission_files_paths))
     else:
         raise SubmissionNotFoundError
 
-def update_submission_files(course, lab, submission_id, submission_files):
+def update_submission_files(course, lab, submission_id, submission_files, submission_key):
```

Comment:
if this function takes multiple submissions then the param should be `submission_keys`

Reply:
it handles a single submission, and for 'files' it's named like that because it may update multiple files in a single submission

Reply:
this is addressed in the other PR